nbbuild/rat-exclusions.txt

```diff
@@ -220,6 +220,7 @@ javafx/javafx2.project/src/org/netbeans/modules/javafx2/project/templates/**
 java/jshell.support/src/org/netbeans/modules/jshell/resources/consoleExample.jsh
 java/junit.ui/src/org/netbeans/modules/junit/ui/resources/*.template
 ide/languages.diff/src/org/netbeans/modules/languages/diff/DiffExample.diff
+ide/languages.go/src/org/netbeans/modules/languages/go/*.go
```

Comment:
Please use the `licenseinfo.xml` to declare `GoExample.go` as `Apache-2.0-ASF`.

Reply:
Done.